docs/tips-n-tricks/Wormholes/gas-harvesting/ninja-huffing.md

```diff
@@ -18,15 +18,15 @@ Only one site cannot be ninja-huffed : the [Ordinary Perimeter Reservoir](https:
 
 ### Ships
 
-All ships and fits can ninja-huff
+All ships and fits can ninja-huff, as it requires no specific gear.
 
 ## How to
 
-Whenever a player first warps to a site (gas site, ore belts, etc), a timer is triggered, at the end of which rats will spawn.
+Whenever a player first warps to a site (gas site, ore belts, etc) **and is uncloaked**, a timer is triggered, at the end of which rats will spawn.
 
 For Wormhole gas sites, that timer is anywhere between 15 and up to 25 minutes. To effectively Ninja huff, simply set a timer (on your phone for example) of 15 minutes right as you warp in. When it rings, keep an eye out for the rats, and be ready to warp out to your safe.
 
-Note that you should always be paying attention to your dscan and be ready to warp out at any time regardless of rats timer.
+Note that you should always be paying attention to your dscan and be ready to warp out at any time regardless of rats timer. Core/Sisters Probes are cause for concern, but not a sure sign - if you see another T1 Explorer on Dscan it is less likely to be cause for alarm. Combat scanner probes, however, are valid reason to Cloak and warp to a Safe. If you have no cloak, pingponging on various astral bodies and your safe can help deter any would-be attacker. Be cautious about warping to your exit hole (you did plan your quick exit, right?) immediately when a threat is spotted, as they may be cloaked on the hole, waiting to bump you off.
```

Comment:
This is very good but here is not the place for it. Maybe this should be in the Basics guide instead.

Reply:
Moved to basics.